src/app/constant.config.ts

```diff
@@ -0,0 +1,10 @@
+
+export const Roles = [
+    { name: 'Content Creator', value: 'CONTENT_CREATOR' },
+    { name: 'Content Reviewer', value: 'CONTENT_REVIEWER' },
+    { name: 'Book Creator', value: 'BOOK_CREATOR' },
+    { name: 'Book Reviewer', value: 'BOOK_REVIEWER' },
+    { name: 'Org Admin', value: 'ORG_ADMIN' },
+    { name: 'public', value: 'PUBLIC' }
```

Comment:
In public 'p' should be capital or small?

Reply:
Resolved